packages/rpc-methods/package.json

```diff
@@ -69,5 +70,11 @@
   "publishConfig": {
     "access": "public",
     "registry": "https://registry.npmjs.org/"
+  },
+  "lavamoat": {
```

Comment:
Any idea why these popped up now?

Reply:
This is because the first time I installed `snaps-ui` it was set to `workspace`. Tested with the npm version and it's not needed, removing.